src/main/java/lottoauto/domain/LottoStatus.java

```diff
@@ -0,0 +1,5 @@
+package lottoauto.domain;
+
+public enum LottoStatus {
+    MatchThree, MatchFour, MatchFive, MatchSix, Nothing;
```

Comment:
```suggestion
    MatchThree(5,000)
```
이를 활용하면 과 같은 표현이 가능해집니다!
외부에 따로 상금 필드를 선언할 필요가 없어지는 것이죠 :)